python/svix/api.py

```diff
@@ -125,15 +133,15 @@ def _api(self) -> t.Generator[ApiClass, None, None]:
 class Authentication(ApiBase[AuthenticationApi]):
     _ApiClass = AuthenticationApi
 
-    def dashboard_access(self, app_id: str) -> DashboardAccessOut:
+    def dashboard_access(self, app_id: str, options: CreateOptions = CreateOptions()) -> DashboardAccessOut:
         with self._api() as api:
             return api.get_dashboard_access_api_v1_auth_dashboard_access_app_id_post(
-                app_id=app_id, _check_return_type=False
+                app_id=app_id, _check_return_type=False, **options.to_dict()
```

Comment:
any reason we aren't doing this instead of making a special `to_dict` function?
```suggestion
                app_id=app_id, _check_return_type=False, **asdict(options)
```

Reply:
I think it's to skip `None` values. I also had the same question in my head. :P

Reply:
Right, asdict does cause an issue when no options are passed

Reply:
Nones are fine since we pass _check_return_type=False no?

Reply:
svix.application.create(ApplicationIn(name="Application name"))


ApiTypeError: Invalid type for variable 'idempotency_key'. Required value type is str and passed type was NoneType at ['idempotency_key']

Reply:
@svix-frank, you created the original code I think (she just copied what's done elsewhere).

Anyhow, what she did is correct, `None` are not a valid value for these.